docs/Membership/Consequences_and_Warnings.md

```diff
@@ -74,4 +74,4 @@ The board will make as much detail about bans and current warnings as they feel
 
 If you have been asked to stop behaving in a certain way, do so. It's fine to ask what it is you're doing that is upsetting the person if it is not clear, but you shouldn't ask why the person is upset by it, it's enough that they are.
 
-If you have to take action to stop someone behaving in a way contrary to the Citizen Hacman  [Membership Agreement](https://moodle.hacman.org.uk/mod/page/view.php?id=38 "Membership Agreement")  do so, if possible consult with other members and/or the board, however, if action is required take it and explain later.
\ No newline at end of file
+If you have to take action to stop someone behaving in a way contrary to the Citizen Hacman  [Membership Agreement](https://moodle.hacman.org.uk/mod/page/view.php?id=38 "Membership Agreement")  do so, if possible consult with other members and/or the board, however, if action is required take it and explain later.
```

Comment:
Thanks for this! Just a note that the link here is out of date as I don't think Moodle is in use. 

Could you change the URL here to be https://hacman.org.uk/rules ?